articles/what-is-an-ssl-certificate.mdx

```diff
@@ -0,0 +1,110 @@
+
+Warnings about unsecure websites can lead to a higher user bounce-rate (where users immediately "bounce" off your site before seeing any of your content). This might be because users are security conscious or because they lose interest due to an inefficient experience.
+
+You can avoid these issues by adding an SSL certificate to your Site to enable HTTPS.
+
+## How do SSL certificates work?
+
+SSL certificates contain the following details about your Site:
+
+* **Domain Name** — The address used to access a website (e.g., [cloudcannon.com](https://cloudcannon.com/)).
+* **Certificate expiration date** — The date a certificate expires. This is normally a year after it was generated.
+* **Public key** — A string used to encrypt data, protecting communication channels from unauthorized access. Data can only be decrypted with the corresponding Private key.
+* **Certificate Chain** — A sequence of certificates which links the SSL certificate for your Site to a trusted certificate authority, providing legitimacy to your SSL certificate.
+
+SSL certificates also come with a private key, which is not shared with the user's browser.
+
+* **Private key** — A string used to decrypt data.
+
+<comp.Notice info_type="important">
+  Do not write down your Private key in a publicly accessible or unsecure place. If you think you the security of your Private key has been compromized, please generate a new SSL certificate.
+</comp.Notice>
+
+When a browser attempts to connect to your website, it uses a process called a *TLS handshake*. This process validates the identity of the web server, the domain of your website, and prevents unauthorized access to the data passing between the user and your website.
+
+In simple terms, the *TLS handshake* includes the following steps:
+
+1. The user's browser contacts the web server hosting your website, specifying what version of TLS the browser is using and a random string called a "Client Random".
+2. The web server replies to the browser with the SSL certificate and a random string called a "Server Random".
+3. The browser will verify the SSL certificate with the certificate authority that issued it, confirming the authenticity of the web server and the domain.
+4. The browser contacts the web server again with a random string encrypted with the Public key from the SSL certificate.
+5. The web server uses the corresponding Private key to decrypt the message.
+6. If the web server and the user's browser agree on the strings for Client Random, Server Random, and the decrypted message, then the web server will allow the browser to access the website in an encrypted session.
+
+This process only takes milliseconds to complete, so it will not affect the user's experience on your website.
+
+SSL certificates last for about a year and should be renewed at yearly intervals to maintain your website's security. You can renew you SSL certificate more often, but most website do not require this level of security.
```

Comment:
Our automatically generated SSL certificates are for 90 days but if they use a custom SSL, it would likely be 1 year. Perhaps we can move this comment to the Custom SSL page instead?